modules/gltf/gltf_document_extension_convert_importer_mesh.cpp

```diff
@@ -29,6 +29,9 @@
 /*************************************************************************/
 
 #include "gltf_document_extension_convert_importer_mesh.h"
+
+#include "modules/gltf/gltf_state.h"
```

Comment:
```suggestion
#include "gltf_state.h"
```

Reply:
Thanks for catching those extra includes.

The PR went through a redesign where I removed an array of gltf document extensions from both the gltf importer plugin and the options array from inside the gltf extension, but the includes didn't get removed.

Also I uses clangd with vscode do you know how to remove the automatic includes?